tests/isr_yield_higher/main.c

```diff
@@ -27,11 +27,13 @@
 #define TEST_TIME (200000U)
 
 static char t2_stack[THREAD_STACKSIZE_MAIN];
+static uint32_t start_time;
 
 static void *second_thread(void *arg)
 {
-    (void) arg;
-    if (xtimer_now_usec() < TEST_TIME) {
+    uint32_t start_time = *((uint32_t *)arg);
```

Comment:
start_time is static so available globally. There's no need to pass it via the thread context.